client/tests/integration/events/data.rs

```diff
@@ -138,7 +138,7 @@ fn transaction_execution_should_produce_events(
     let listener = client.clone();
     let (init_sender, init_receiver) = mpsc::channel();
     let (event_sender, event_receiver) = mpsc::channel();
-    let event_filter = DataEventFilter::AcceptAll.into();
+    let event_filter = DataEventFilter::Any;
```

Comment:
since you didn't like renaming `new` to `any`, why then don't we use like elsewhere:

```suggestion
    let event_filter = DataEventFilter::new();
```